flink-staging/flink-ml/src/main/scala/org/apache/flink/ml/pipeline/Predictor.scala

```diff
@@ -35,7 +35,7 @@ import org.apache.flink.ml.common.{FlinkMLTools, ParameterMap, WithParameters}
   *
   * @tparam Self Type of the implementing class
   */
-trait Predictor[Self] extends Estimator[Self] with WithParameters with Serializable {
+trait Predictor[Self] extends Estimator[Self] with WithParameters {
```

Comment:
The `Estimator` types should never be shipped to the TaskManager. Usually they contain some state in the form of other `DataSets` which aren't serializable anyways. However, the operations have to be `Serializable`, because they can contain values which are referenced by the Flink operations.